docs/site/BelongsTo-relation.md

```diff
@@ -8,6 +8,14 @@ permalink: /doc/en/lb4/BelongsTo-relation.html
 
 ## Overview
 
+{% include note.html content="
+This relation best works with databases that support foreign key
+constraint (SQL).
+Using this relation with NoSQL databases will result in unexpected behavior,
+such as the ability to create a relation with a model that does not exist. We are working on a solution to better handle this. It is fine to use this relation with NoSQL databases for purposes such as navigating
```

Comment:
`We are working on a solution to better handle this` is there a GitHub issue we can link them to?

Reply:
AFAIK, there is none. An issue was opened to initiate the discussion on the matter - https://github.com/strongloop/loopback-next/issues/2127 - it is closed now. @bajtos, you thougts?

Reply:
Let's use the following issue for the link: _Docs: what relation types will work well with my database?_ #2341